### DISCUSSION TITLE
How to use tailwindcss JIT in alephjs

### DISCUSSION THREAD

**Discussion by dfang** [alephjs/aleph.js#400] [Q&A]
what is the best/right way to use tailwindcss in alephjs ?

`tailwindcss` jit mode and plugins like  `@tailwindcss/typography`, `@tailwindcss/forms`, `@tailwindcss/line-clamp`, `@tailwindcss/aspect-ratio`.

do i have to use windicss plugin for tailwindcss?

- https://alephjs.org/docs/basic-features/built-in-css-support#postcss
- https://alephjs.org/docs/plugins/community-plugins
- https://alephjs.org/docs/api-reference/plugin-api#tailwind-jit-for-jsx

**Comment by ije** [comment]
windicss's API is very simple to create an aleph, i tried to create a tailwind plugin for alephjs but failed...
